cfme/tests/infrastructure/test_vm_reconfigure.py

```diff
@@ -191,3 +191,303 @@ def test_vm_reconfig_add_remove_hw_hot(provider, small_vm, ensure_vm_running, ch
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+@pytest.mark.parametrize('disk_type', ['thin', 'thick'])
+@pytest.mark.parametrize(
+    'disk_mode', ['persistent', 'independent_persistent', 'independent_nonpersistent'])
+def test_vm_reconfig_resize_disk_hot(disk_type, disk_mode):
+    """
+    Polarion:
+        assignee: nansari
+        initialEstimate: 1/6h
+        caseimportance: high
+        caselevel: integration
+        caseposneg: positive
+        caseautomation: Non Automated
+        testtype: functional
+        startsin: 5.9
+        casecomponent: infra
+        testSteps:
+            1. Resize the disk while VM is running
+            2. Go to Compute -> infrastructure -> Virtual Machines -> Select Vm
+            3. Go to VM reconfiguration
+            4. Click on Add Disk -> select disk_type and disk_mode , save and submit
+            5. Go to VM reconfiguration and resize the disk
+            6. Check the changes in VM reconfiguration page
+            7. Remove the disk
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+@pytest.mark.parametrize(
+    'adapters_type', ['DPortGroup', 'VmNetwork', 'MgmtNetwork', 'VmKernel'])
+def test_vm_reconfig_add_remove_network_adapters(adapters_type):
+    """
+    Polarion:
+        assignee: nansari
+        initialEstimate: 1/6h
+        caseimportance: high
+        caselevel: integration
+        caseposneg: positive
+        caseautomation: Non Automated
+        testtype: functional
+        startsin: 5.9
+        casecomponent: infra
+        testSteps:
+            1. Go to Compute -> infrastructure -> Virtual Machines -> Select Vm
+            2. Go to VM reconfiguration
+            3. Click on Add Adapters -> select type , save and submit
+            4. Check the changes in VM reconfiguration page
+            5. Remove the Adapters
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_reconfigure_vm_vmware_mem_multiple():
+    """
+    Polarion:
+        assignee: nansari
+        initialEstimate: 1/6h
+        caseimportance: medium
+        caselevel: integration
+        caseposneg: positive
+        caseautomation: Non Automated
+        testtype: functional
+        startsin: 5.6
+        casecomponent: infra
+        testSteps:
+            1. get new configured appliance ->add vmware provider
+            2. provision 2 new vms
+            3. power off 1 vm -> select both vms
+            4. configure-->reconfigure vm
+            5. power on vm
+            6. check changes
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+def test_vm_reconfig_attach_iso_vsphere67_nested():
+    """
+
+    Bugzillas:
+        * 1533728
+
+    Polarion:
+        assignee: nansari
+        initialEstimate: 1/6h
+        caseimportance: high
+        caselevel: integration
+        caseposneg: positive
+        caseautomation: Non Automated
+        testtype: functional
+        startsin: 5.10
```

Comment:
if it starts in `5.10`, there should be a decorator `@pytest.mark.ignore_stream('5.9')`